kernel-modules/MODULE_VERSION

```diff
@@ -1 +1 @@
-2.9.0-rc2
+2.9.0-rc4
```

Comment:
why do we need a module version bump for this?

Reply:
I was under the impression it wont pickup Falco changes otherwise. You're saying it's not correct?

Reply:
a bump is only necessary if there is a change to the drivers, so since this is just userspace we don't need to :)